kodi-wayland not linking on Ubuntu 25.04 due undefined reference to 'inflateEnd'

<!--- Please fill out this template to the best of your ability. You can always edit this issue once you have created it. -->
<!--- Read the following link before you create a new problem report: https://kodi.wiki/view/HOW-TO:Submit_a_bug_report  -->
## Bug report
### Describe the bug
Here is a clear and concise description of what the problem is:
<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->
<!--- A bug report that is not clear will be closed -->
<!--- Put your text below this line -->
I am trying to compile kodi on Ubuntu 25.04 (amd64).

kodi-wayland binary will fail to link with:
```
build/filesystem/filesystem.a(ZipFile.cpp.o):ZipFile.cpp:function XFILE::CZipFile::Close():(.text+0x2ea): error: undefined reference to 'inflateEnd'
```


## Expected Behavior
Here is a clear and concise description of what was expected to happen:
<!--- Tell us what should happen -->
<!--- Put your text below this line -->
To compile and produce a binary


## Actual Behavior
<!--- Tell us what happens instead -->
<!--- Put your text below this line -->
Fails to compile

## Possible Fix
<!--- Not obligatory, but suggest a fix or reason for the bug -->
<!--- Put your text below this line -->
No idea


### To Reproduce
Steps to reproduce the behavior:
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->
<!--- Put your text below this line -->
1. Ubuntu 25.04 amd64 install
2. Install requested packages according to docs for building kodi on Linux
3. Clone repo
4. mkdir $HOME/kodi-build
5. cd $HOME/kodi-build
6. cmake ../kodi -DCMAKE_INSTALL_PREFIX=/usr/local -DCORE_PLATFORM_NAME=wayland -DAPP_RENDER_SYSTEM=gl
7. cmake --build . -- VERBOSE=1 -j$(getconf _NPROCESSORS_ONLN)


### Debuglog
<!--- Put your text below this line -->
<!--- A debuglog is always mandatory when creating an issue. Provide one! -->
The debuglog can be found here:
cmake ../kodi -DCMAKE_INSTALL_PREFIX=/usr/local -DCORE_PLATFORM_NAME=wayland -DAPP_RENDER_SYSTEM=gl
```
-- The CXX compiler identification is GNU 14.2.0
-- The C compiler identification is GNU 14.2.0
-- The ASM compiler identification is GNU
-- Found assembler: /usr/bin/cc
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Mirror download location: http://mirrors.kodi.tv
-- Source directory: /home/defiant/Applications/kodi
-- Build directory: /home/defiant/Applications/kodi-build
-- Generator: Single-configuration: Release (Unix Makefiles)
-- CMake Version: 3.31.6
-- System type: Linux
-- Linker: GNU gold
-- Host architecture is little-endian
-- Core system type: linux
-- Platform: wayland
-- CPU: x86_64, ARCH: x86_64-linux
-- Cross-Compiling: FALSE
-- Execute build artefacts on host: 
-- Depends based build: 
-- statx is available
-- Checking to see if CXX compiler accepts flag -msse
-- Checking to see if CXX compiler accepts flag -msse - yes
-- Checking to see if CXX compiler accepts flag -msse2
-- Checking to see if CXX compiler accepts flag -msse2 - yes
-- Checking to see if CXX compiler accepts flag -msse3
-- Checking to see if CXX compiler accepts flag -msse3 - yes
-- Checking to see if CXX compiler accepts flag -mssse3
-- Checking to see if CXX compiler accepts flag -mssse3 - yes
-- Checking to see if CXX compiler accepts flag -msse4.1
-- Checking to see if CXX compiler accepts flag -msse4.1 - yes
-- Checking to see if CXX compiler accepts flag -msse4.2
-- Checking to see if CXX compiler accepts flag -msse4.2 - yes
-- Checking to see if CXX compiler accepts flag -mavx
-- Checking to see if CXX compiler accepts flag -mavx - yes
-- Checking to see if CXX compiler accepts flag -mavx2
-- Checking to see if CXX compiler accepts flag -mavx2 - yes
-- Found SSE: 1
-- Found SSE2: 1
-- Found SSE3: 1
-- Found SSSE3: 1
-- Found SSE4_1: 1
-- Found SSE4_2: 1
-- Found AVX: 1
-- Found AVX2: 1
-- Could NOT find CCache (missing: CCACHE_PROGRAM) 
-- Could NOT find ClangFormat (missing: CLANG_FORMAT_EXECUTABLE) 
-- Found FlatC Compiler: /bin/flatc (found version "23.5.26")
-- Found JsonSchemaBuilder: /home/defiant/Applications/kodi-build/build/bin/JsonSchemaBuilder
-- Building internal TexturePacker
-- Shipping internal TexturePacker
-- Internal TexturePacker will be executed during build
-- Found Alsa: /usr/lib/x86_64-linux-gnu/libasound.so (found suitable version "1.2.13", minimum required is "1.0.27")
-- Found Avahi: /usr/lib/x86_64-linux-gnu/libavahi-client.so (found version "0.8")
-- Found Bluetooth: /usr/lib/x86_64-linux-gnu/libbluetooth.so
-- Found Bluray: /usr/lib/x86_64-linux-gnu/libbluray.so (found suitable version "1.3.4", minimum required is "0.9.3")
-- Found CAP: /usr/lib/x86_64-linux-gnu/libcap.so (found version "2.73")
-- Found CEC: /usr/lib/x86_64-linux-gnu/libcec.so (found suitable version "6.0.2", minimum required is "4.0.0")
-- Could NOT find Dav1d (missing: DAV1D_LIBRARY DAV1D_INCLUDE_DIR) 
-- Found DBus: /usr/lib/x86_64-linux-gnu/libdbus-1.so (found version "1.16.2")
CMake Warning at cmake/modules/FindCdio.cmake:26 (message):
  Detected libcdio (2.2.0) and libcdio++ () version mismatch.  libcdio++ will
  not be used.
Call Stack (most recent call first):
  cmake/modules/FindIso9660pp.cmake:16 (find_package)
  cmake/scripts/common/Macros.cmake:376 (find_package)
  cmake/scripts/common/Macros.cmake:426 (find_package_with_ver)
  CMakeLists.txt:285 (core_optional_dep)


-- Could NOT find Iso9660pp (missing: ISO9660PP_LIBRARY ISO9660PP_INCLUDE_DIR) (Required is at least version "2.1.0")
-- Found LCMS2: /usr/lib/x86_64-linux-gnu/liblcms2.so (found suitable version "2.16", minimum required is "2.10")
-- Found LircClient: /usr/lib/x86_64-linux-gnu/liblirc_client.so
-- Could NOT find MDNS (missing: MDNS_LIBRARY MDNS_INCLUDE_DIR) 
-- Found MicroHttpd: /usr/lib/x86_64-linux-gnu/libmicrohttpd.so (found suitable version "1.0.1", minimum required is "0.9.40")
-- Found NFS: /usr/lib/x86_64-linux-gnu/libnfs.so (found suitable version "5.0.2", minimum required is "3.0.0")
-- Could NOT find Pipewire (missing: PIPEWIRE_LIBRARY PIPEWIRE_INCLUDE_DIR SPA_INCLUDE_DIR) (Required is at least version "0.3.50")
-- Found Plist: /usr/lib/x86_64-linux-gnu/libplist-2.0.so (found version "2.6.0")
-- Found PulseAudio: /usr/lib/x86_64-linux-gnu/libpulse.so (found suitable version "17.0", minimum required is "11.0.0")
-- Found SmbClient: /usr/lib/x86_64-linux-gnu/libsmbclient.so (found version "0.8.0")
-- Found Sndio: /usr/lib/x86_64-linux-gnu/libsndio.so
-- Found UDEV: /usr/lib/x86_64-linux-gnu/libudev.so (found version "257")
-- Found Udfread: /usr/lib/x86_64-linux-gnu/libudfread.so (found version "1.1.2")
-- Found XSLT: /usr/lib/x86_64-linux-gnu/libxslt.so (found version "1.1.39")
-- Found VAAPI: /usr/lib/x86_64-linux-gnu/libva.so (found suitable version "1.22.0", minimum required is "0.39.0")
-- Found ASS: /usr/lib/x86_64-linux-gnu/libass.so (found suitable version "0.17.3", minimum required is "0.15.0")
-- Found CrossGUID: optimized;/home/defiant/Applications/kodi-build/build/lib/libcrossguid.a;debug;/home/defiant/Applications/kodi-build/build/lib/libcrossguid-dgb.a (found version "ca1bf4b810e2d188d04cb6286f957008ee1b7681")
-- Found Curl: /usr/lib/x86_64-linux-gnu/libcurl.so (found version "8.12.1")
-- Found Exiv2: /usr/lib/x86_64-linux-gnu/libexiv2.so.0.28.5 (found version "0.28.5")
-- Checking for modules 'libavcodec>=61.3.100;libavfilter>=10.1.100;libavformat>=61.1.100;libavutil>=59.8.100;libswscale>=8.1.100;libswresample>=5.1.100;libpostproc>=58.1.100'
--   Found libavcodec, version 61.19.101
--   Found libavfilter, version 10.4.100
--   Found libavformat, version 61.7.100
--   Found libavutil, version 59.39.100
--   Found libswscale, version 8.3.100
--   Found libswresample, version 5.3.100
--   Found libpostproc, version 58.3.100
-- Found FFMPEG: /usr/include/x86_64-linux-gnu (found version "7.0.0")
-- Found FlatBuffers: /usr/include/ (found version "23.3.3")
-- Found Fmt: /usr/lib/x86_64-linux-gnu/libfmt.so.10.1.0
-- Found FreeType: /usr/lib/x86_64-linux-gnu/libfreetype.so (found version "26.2.20")
-- Found FriBidi: /usr/lib/x86_64-linux-gnu/libfribidi.so (found version "1.0.16")
-- Found Fstrcmp: /usr/lib/libfstrcmp.so (found version "0.7.D001")
-- Found HarfBuzz: /usr/lib/x86_64-linux-gnu/libharfbuzz.so (found version "10.2.0")
-- Performing Test Iconv_IS_BUILT_IN
-- Performing Test Iconv_IS_BUILT_IN - Success
-- Found LibDvdNav: /home/defiant/Applications/kodi-build/build/lib/libdvdnav.a (found version "6.1.1-Next-Nexus-Alpha2-2")
-- Found Lzo2: /usr/lib/x86_64-linux-gnu/liblzo2.so
-- Found PCRE2: /usr/lib/x86_64-linux-gnu/libpcre2-8.so (found version "10.45")
-- Found NlohmannJSON: /usr/include (found suitable version "3.11.3", minimum required is "3.2.0")
-- Found Threads: TRUE
-- Found Spdlog: /usr/lib/x86_64-linux-gnu/libspdlog.so.1.15.1
-- Found Sqlite3: /usr/lib/x86_64-linux-gnu/libsqlite3.so (found version "3.46.1")
-- Found TagLib: /usr/lib/x86_64-linux-gnu/libtag.so.2.0.2 (Required is at least version "1.9.0")
-- Found TinyXML: /usr/lib/x86_64-linux-gnu/libtinyxml.so (found version "2.6.2")
-- Found TinyXML2: /usr/lib/x86_64-linux-gnu/libtinyxml2.so.10.0.0
-- Found ZLIB: /usr/lib/x86_64-linux-gnu/libz.so (found version "1.3.1")
-- Found WaylandProtocols: 1 (found suitable version "1.41", minimum required is "1.7")
-- Found Waylandpp: /usr/include (Required is at least version "0.2.2")
-- Found LibDRM: /usr/lib/x86_64-linux-gnu/libdrm.so (found suitable version "2.4.124", minimum required is "2.4.95")
-- Found Xkbcommon: /usr/lib/x86_64-linux-gnu/libxkbcommon.so (found suitable version "1.7.0", minimum required is "0.4.1")
-- Found OpenGl: /usr/lib/x86_64-linux-gnu/libGL.so
-- Found EGL: /usr/lib/x86_64-linux-gnu/libEGL.so (found version "1.5")
-- Could NOT find MariaDBClient (missing: MARIADBCLIENT_LIBRARY MARIADBCLIENT_INCLUDE_DIR) 
-- Found MySqlClient: /usr/lib/x86_64-linux-gnu/libmysqlclient.so (found version "8.4.5")
-- #---- CONFIGURATION ----#
-- Platforms: wayland
-- App package: org.xbmc.kodi
-- -- PATH config --
-- Prefix: /usr/local
-- Libdir: /usr/local/lib
-- Bindir: /usr/local/bin
-- Includedir: /usr/local/include
-- Datarootdir: /usr/local/share
-- Datadir: /usr/local/share
-- Docdir: /usr/local/share/doc/kodi
-- CCACHE enabled: No
-- CLANGFORMAT enabled: No
-- CLANGTIDY enabled: No
-- CPPCHECK enabled: No
-- INCLUDEWHATYOUUSE enabled: No
-- ALSA enabled: Yes
-- AVAHI enabled: Yes
-- BLUETOOTH enabled: Yes
-- BLURAY enabled: Yes
-- CAP enabled: Yes
-- CEC enabled: Yes
-- DAV1D enabled: No
-- DBUS enabled: Yes
-- ISO9660PP enabled: No
-- LCMS2 enabled: Yes
-- LIRCCLIENT enabled: Yes
-- MDNS enabled: No
-- MICROHTTPD enabled: Yes
-- NFS enabled: Yes
-- PIPEWIRE enabled: No
-- PLIST enabled: Yes
-- PULSEAUDIO enabled: Yes
-- PYTHON enabled: Yes
-- SMBCLIENT enabled: Yes
-- SNDIO enabled: Yes
-- UDEV enabled: Yes
-- UDFREAD enabled: Yes
-- XSLT enabled: Yes
-- VAAPI enabled: Yes
-- MARIADBCLIENT enabled: No
-- MYSQLCLIENT enabled: Yes
-- Configuring done (37.7s)
-- Generating done (2.2s)
-- Build files have been written to: /home/defiant/Applications/kodi-build
```

cmake --build . -- VERBOSE=1 -j$(getconf _NPROCESSORS_ONLN)
```
[100%] Building CXX object CMakeFiles/kodi.dir/xbmc/platform/posix/main.cpp.o
/usr/bin/c++ -DCMS_NO_REGISTER_KEYWORD -DEXV_LOCALEDIR=\"../share/locale\" -DFFMPEG_VER_SHA=7.0.0 -DFMT_SHARED -DHAS_AIRPLAY -DHAS_AIRTUNES -DHAS_ALSA -DHAS_AVAHI -DHAS_DBUS -DHAS_EGL -DHAS_FILESYSTEM_NFS -DHAS_FILESYSTEM_SMB -DHAS_GL -DHAS_LIRC -DHAS_MYSQL -DHAS_NFS_MOUNT_GETEXPORTS_TIMEOUT -DHAS_PULSEAUDIO -DHAS_PYTHON -DHAS_SNDIO -DHAS_UDFREAD -DHAS_WEB_INTERFACE -DHAS_WEB_SERVER -DHAS_ZEROCONF -DHAVE_DRM_MODIFIER_NAME -DHAVE_EGLEXTANGLE -DHAVE_HDR_OUTPUT_METADATA -DHAVE_LCMS2 -DHAVE_LIBBLUETOOTH -DHAVE_LIBBLURAY -DHAVE_LIBBLURAY_BDJ -DHAVE_LIBCAP -DHAVE_LIBCEC -DHAVE_LIBUDEV -DHAVE_LIBVA -DHAVE_LIBXSLT -DHAVE_NEW_CROSSGUID -DHAVE_WAYLAND -DNPT_CONFIG_ENABLE_LOGGING -DSPDLOG_COMPILED_LIB -DSPDLOG_FMT_EXTERNAL -DSPDLOG_SHARED_LIB -DTINYXML2_IMPORT -D_FILE_OFFSET_BITS=64 -Dkodi_EXPORTS -I/home/defiant/Applications/kodi-build -I/home/defiant/Applications/kodi -I/home/defiant/Applications/kodi/lib -I/home/defiant/Applications/kodi/xbmc -I/home/defiant/Applications/kodi/xbmc/platform/linux -I/home/defiant/Applications/kodi/xbmc/cores/VideoPlayer -I/home/defiant/Applications/kodi-build/build -I/home/defiant/Applications/kodi/xbmc/platform/posix -isystem /home/defiant/Applications/kodi-build/build/include -isystem /usr/include/freetype2 -isystem /usr/include/libpng16 -isystem /usr/include/harfbuzz -isystem /usr/include/glib-2.0 -isystem /usr/lib/x86_64-linux-gnu/glib-2.0/include -isystem /usr/include/sysprof-6 -isystem /usr/include/fribidi -isystem /usr/include/p11-kit-1 -isystem /usr/include/mit-krb5 -isystem /home/defiant/Applications/kodi-build/build/cores/RetroPlayer/messages -isystem /usr/include/lzo -isystem /usr/include/taglib -isystem /usr/include/libdrm -isystem /usr/include/libxml2 -isystem /usr/include/libcec -isystem /usr/include/dbus-1.0 -isystem /usr/lib/x86_64-linux-gnu/dbus-1.0/include -isystem /usr/include/python3.13 -isystem /usr/include/samba-4.0 -isystem /home/defiant/Applications/kodi/lib/libUPnP -isystem /home/defiant/Applications/kodi/lib/libUPnP/Platinum/Source/Core -isystem /home/defiant/Applications/kodi/lib/libUPnP/Platinum/Source/Platinum -isystem /home/defiant/Applications/kodi/lib/libUPnP/Platinum/Source/Devices/MediaConnect -isystem /home/defiant/Applications/kodi/lib/libUPnP/Platinum/Source/Devices/MediaRenderer -isystem /home/defiant/Applications/kodi/lib/libUPnP/Platinum/Source/Devices/MediaServer -isystem /home/defiant/Applications/kodi/lib/libUPnP/Neptune/Source/Core -isystem /home/defiant/Applications/kodi/lib/libUPnP/Neptune/Source/System/Posix -Wall -Wdouble-promotion -Wmissing-field-initializers -Wsign-compare -Wextra -Wno-unused-parameter -Wnon-virtual-dtor -O3 -DNDEBUG -std=c++20 -DTARGET_POSIX -DTARGET_LINUX -D_GNU_SOURCE -DHAVE_LINUX_UDMABUF=1 -DHAVE_LINUX_DMA_HEAP=1 -DHAVE_LINUX_DMA_BUF=1 -DHAVE_MKOSTEMP=1 -DHAVE_GETMNTENT_R=1 -DHAVE_LINUX_MEMFD=1 -DHAVE_STATX=1 -DHAVE_SSE=1 -DHAVE_SSE2=1 -DHAVE_SSE3=1 -DHAVE_SSSE3=1 -DHAVE_SSE4_1=1 -D__STDC_CONSTANT_MACROS -D_FILE_OFFSET_BITS=64 -DHAS_POSIX_NETWORK -DHAS_LINUX_NETWORK -DHAS_BUILTIN_SYNC_ADD_AND_FETCH=1 -DHAS_BUILTIN_SYNC_SUB_AND_FETCH=1 -DHAS_BUILTIN_SYNC_VAL_COMPARE_AND_SWAP=1 -DHAVE_INOTIFY=1 -DHAVE_POSIX_FADVISE=1 -DHAVE_LOCALTIME_R=1 -DHAVE_GMTIME_R=1 -DHAVE_INTTYPES_H=1 -DHAS_UPNP=1 -DHAS_OPTICAL_DRIVE -DHAS_CDDA_RIPPER -DBIN_INSTALL_PATH=\"/usr/local/lib/kodi\" -DINSTALL_PATH=\"/usr/local/share/kodi\" -Werror=double-promotion -Werror=missing-field-initializers -Werror=sign-compare -pthread -MD -MT CMakeFiles/kodi.dir/xbmc/platform/posix/main.cpp.o -MF CMakeFiles/kodi.dir/xbmc/platform/posix/main.cpp.o.d -o CMakeFiles/kodi.dir/xbmc/platform/posix/main.cpp.o -c /home/defiant/Applications/kodi/xbmc/platform/posix/main.cpp
[100%] Linking CXX executable kodi-wayland
/usr/bin/cmake -E cmake_link_script CMakeFiles/kodi.dir/link.txt --verbose=1
/usr/bin/c++ -Wall -Wdouble-promotion -Wmissing-field-initializers -Wsign-compare -Wextra -Wno-unused-parameter -Wnon-virtual-dtor -O3 -DNDEBUG  -fuse-ld=gold -Wl,--export-dynamic -rdynamic -Wl,--dependency-file=CMakeFiles/kodi.dir/link.d CMakeFiles/kodi.dir/xbmc/platform/posix/main.cpp.o -o kodi-wayland  -Wl,--whole-archive build/network/httprequesthandler/python/network_httprequesthandlers_python.a build/network/httprequesthandler/network_httprequesthandlers.a build/network/upnp/network_upnp.a build/interfaces/python/python_interface.a build/interfaces/legacy/wsgi/legacy_interface_wsgi.a build/interfaces/legacy/legacy_interface.a build/rendering/gl/rendering_gl.a build/network/dacp/network_dacp.a build/cdrip/cdrip.a build/platform/linux/network/zeroconf/platform_linux_network_zeroconf.a build/windowing/linux/windowing_linux.a build/platform/posix/utils/platform_posix_utils.a build/platform/posix/threads/platform_posix_threads.a build/platform/posix/storage/discs/platform_posix_storage_discs.a build/platform/posix/network/platform_posix_network.a build/platform/posix/filesystem/platform_posix_filesystem.a build/platform/posix/platform_posix.a build/platform/linux/threads/platform_linux_threads.a build/platform/linux/storage/platform_linux_storage.a build/platform/linux/powermanagement/platform_linux_powermanagement.a build/platform/linux/peripherals/platform_linux_peripherals.a build/platform/linux/network/platform_linux_network.a build/platform/linux/input/input_linux.a build/platform/linux/linuxsupport.a build/platform/common/speech/platform_common_speech.a build/input/touch/generic/input_touch_generic.a build/input/touch/input_touch.a build/cores/VideoPlayer/videorenderers/shaders/videoshaders.a build/cores/VideoPlayer/videorenderers/hwdec/videorenderers_hwdec.a build/cores/VideoPlayer/videorenderers/videorenderers.a build/cores/VideoPlayer/process/process.a build/cores/VideoPlayer/subtitles/webvtt/subtitles_webvtt.a build/cores/VideoPlayer/subtitles/dvdsubtitles.a build/cores/VideoPlayer/inputstreams/dvdinputstreams.a build/cores/VideoPlayer/demuxers/dvddemuxers.a build/cores/VideoPlayer/codecs/video/dvdvideocodecs.a build/cores/VideoPlayer/codecs/overlay/dvdoverlaycodecs.a build/cores/VideoPlayer/codecs/audio/dvdaudiocodecs.a build/cores/VideoPlayer/codecs/dvdcodecs.a build/cores/VideoPlayer/Buffers/videoplayer-buffers.a build/cores/VideoPlayer/VideoPlayer.a build/video/windows/video_windows.a build/video/tags/video_tags.a build/video/jobs/video_jobs.a build/video/guilib/video_guilib.a build/video/dialogs/video_dialogs.a build/video/video.a build/windows/windows.a build/windowing/windowing.a build/weather/weather.a build/view/view.a build/utils_guilib/utils_guilib.a build/utils/utils.a build/threads/threads.a build/storage/storage.a build/rendering/rendering.a build/programs/programs.a build/powermanagement/powermanagement.a build/playlists/playlists.a build/platform/platform_common.a build/pictures_metadata/pictures_metadata.a build/pictures/pictures.a build/messagingHelpers/messagingHelpers.a build/messaging/messaging.a build/imagefiles/imagefiles.a build/guilib_listproviders/guilib_listproviders.a build/guilib_announcement_handlers/guilib_announcement_handlers.a build/guilib_guiinfo/guilib_guiinfo.a build/guilib/guilib.a build/favourites/favourites.a build/dialogs/dialogs.a build/dbwrappers/dbwrappers.a build/commons/commons.a build/application/application.a build/addons_settings/addons_settings.a build/addons_interfaces_gui_dialogs/addons_interfaces_gui_dialogs.a build/addons_interfaces_gui_controls/addons_interfaces_gui_controls.a build/addons_interfaces_gui/addons_interfaces_gui.a build/addons_interfaces/addons_interfaces.a build/addons_gui_skin/addons_gui_skin.a build/addons_gui/addons_gui.a build/addons_binary-addons/addons_binary-addons.a build/addons_addoninfo/addons_addoninfo.a build/addons/addons.a build/xbmc/xbmc.a build/settings/windows/settings_windows.a build/settings/lib/settings_lib.a build/settings/dialogs/settings_dialogs.a build/settings/settings.a build/cores/RetroPlayer/streams/memory/retroplayer_memory.a build/cores/RetroPlayer/streams/retroplayer_streams.a build/cores/RetroPlayer/savestates/retroplayer_savestates.a build/cores/RetroPlayer/rendering/VideoRenderers/rp-videorenderers.a build/cores/RetroPlayer/rendering/rp-rendering.a build/cores/RetroPlayer/process/rp-process.a build/cores/RetroPlayer/playback/retroplayer_playback.a build/cores/RetroPlayer/guiwindows/retroplayer_guiwindows.a build/cores/RetroPlayer/guiplayback/retroplayer_guiplayback.a build/cores/RetroPlayer/guicontrols/retroplayer_guicontrols.a build/cores/RetroPlayer/guibridge/retroplayer_guibridge.a build/cores/RetroPlayer/cheevos/retroplayer_cheevos.a build/cores/RetroPlayer/buffers/video/rp-buffers-video.a build/cores/RetroPlayer/buffers/rp-buffers.a build/cores/RetroPlayer/audio/rp_audio.a build/cores/RetroPlayer/retroplayer.a build/pvr/windows/pvr_windows.a build/pvr/utils/pvr_utils.a build/pvr/timers/pvr_timers.a build/pvr/settings/pvr_settings.a build/pvr/recordings/pvr_recordings.a build/pvr/providers/pvr_providers.a build/pvr/guilib/guiinfo/pvr_guilib_guiinfo.a build/pvr/guilib/pvr_guilib.a build/pvr/filesystem/pvr_filesystem.a build/pvr/epg/pvr_epg.a build/pvr/dialogs/pvr_dialogs.a build/pvr/channels/pvr_channels.a build/pvr/addons/pvr_addons.a build/pvr/pvr.a build/profiles/windows/profiles_windows.a build/profiles/dialogs/profiles_dialogs.a build/profiles/profiles.a build/peripherals/events/peripherals_events.a build/peripherals/dialogs/peripherals_dialogs.a build/peripherals/devices/peripherals_devices.a build/peripherals/bus/virtual/peripheral_bus_virtual.a build/peripherals/bus/peripherals_bus.a build/peripherals/addons/peripherals_addons.a build/peripherals/peripherals.a build/network/websocket/network_websockets.a build/network/network.a build/music/windows/music_windows.a build/music/tags/music_tags.a build/music/jobs/music_jobs.a build/music/infoscanner/music_infoscanner.a build/music/dialogs/music_dialogs.a build/music/music.a build/drm/drm.a build/media/media.a build/interfaces/json-rpc/jsonrpc_interface.a build/interfaces/info/info_interface.a build/interfaces/generic/generic_interface.a build/interfaces/builtins/interfaces_builtins.a build/interfaces/interfaces.a build/input/mouse/generic/input_mouse_generic.a build/input/mouse/input_mouse.a build/input/keymaps/touch/input_keymaps_touch.a build/input/keymaps/remote/input_keymaps_remote.a build/input/keymaps/keyboard/input_keymaps_keyboard.a build/input/keymaps/joysticks/input_keymaps_joysticks.a build/input/keymaps/generic/input_keymaps_generic.a build/input/keymaps/input_keymaps.a build/input/keyboard/generic/input_keyboard_generic.a build/input/keyboard/input_keyboard.a build/input/joysticks/mapping/input_joystick_mapping.a build/input/joysticks/generic/input_joystick_generic.a build/input/joysticks/dialogs/input_joystick_dialogs.a build/input/joysticks/input_joystick.a build/input/actions/input_actions.a build/input/input.a build/games/windows/gameswindows.a build/games/tags/gametags.a build/games/ports/windows/games_ports_windows.a build/games/ports/types/games_ports_types.a build/games/ports/input/games_ports_input.a build/games/ports/guicontrols/games_ports_guicontrols.a build/games/dialogs/osd/gameosddialogs.a build/games/dialogs/gamedialogs.a build/games/controllers/windows/games_controller_windows.a build/games/controllers/types/games_controller_types.a build/games/controllers/listproviders/games_controller_listproviders.a build/games/controllers/input/games_controller_input.a build/games/controllers/guicontrols/games_controller_guicontrols.a build/games/controllers/dialogs/games_controller_dialogs.a build/games/controllers/games_controller.a build/games/agents/windows/games_agents_windows.a build/games/agents/input/games_agents_input.a build/games/addons/streams/game_addon_streams.a build/games/addons/input/gameinput.a build/games/addons/cheevos/gamecheevos.a build/games/addons/gameaddons.a build/games/games.a build/filesystem/videodatabase/videodatabasedirectory.a build/filesystem/musicdatabase/musicdatabasedirectory.a build/filesystem/filesystem.a build/events/windows/events_windows.a build/events/events.a build/cores/playercorefactory/playercorefactory.a build/cores/paplayer/paplayer.a build/cores/externalplayer/externalplayer.a build/cores/dll-loader/exports/util/exports_utils.a build/cores/dll-loader/exports/dllexports.a build/cores/dll-loader/dllloader.a build/cores/audioengine/audioengine.a build/cores/cores.a build/windowing/wayland/windowing_WAYLAND.a build/cores/VideoPlayer/Process/wayland/processWayland.a build/cores/RetroPlayer/process/wayland/rp-process-wayland.a -Wl,--no-whole-archive libkodi.a build/network/httprequesthandler/python/network_httprequesthandlers_python.a build/network/httprequesthandler/network_httprequesthandlers.a build/network/upnp/network_upnp.a build/interfaces/python/python_interface.a build/interfaces/legacy/wsgi/legacy_interface_wsgi.a build/interfaces/legacy/legacy_interface.a build/rendering/gl/rendering_gl.a build/network/dacp/network_dacp.a build/cdrip/cdrip.a build/platform/linux/network/zeroconf/platform_linux_network_zeroconf.a build/windowing/linux/windowing_linux.a build/platform/posix/utils/platform_posix_utils.a build/platform/posix/threads/platform_posix_threads.a build/platform/posix/storage/discs/platform_posix_storage_discs.a build/platform/posix/network/platform_posix_network.a build/platform/posix/filesystem/platform_posix_filesystem.a build/platform/posix/platform_posix.a build/platform/linux/threads/platform_linux_threads.a build/platform/linux/storage/platform_linux_storage.a build/platform/linux/powermanagement/platform_linux_powermanagement.a build/platform/linux/peripherals/platform_linux_peripherals.a build/platform/linux/network/platform_linux_network.a build/platform/linux/input/input_linux.a build/platform/linux/linuxsupport.a build/platform/common/speech/platform_common_speech.a build/input/touch/generic/input_touch_generic.a build/input/touch/input_touch.a build/cores/VideoPlayer/videorenderers/shaders/videoshaders.a build/cores/VideoPlayer/videorenderers/hwdec/videorenderers_hwdec.a build/cores/VideoPlayer/videorenderers/videorenderers.a build/cores/VideoPlayer/process/process.a build/cores/VideoPlayer/subtitles/webvtt/subtitles_webvtt.a build/cores/VideoPlayer/subtitles/dvdsubtitles.a build/cores/VideoPlayer/inputstreams/dvdinputstreams.a build/cores/VideoPlayer/demuxers/dvddemuxers.a build/cores/VideoPlayer/codecs/video/dvdvideocodecs.a build/cores/VideoPlayer/codecs/overlay/dvdoverlaycodecs.a build/cores/VideoPlayer/codecs/audio/dvdaudiocodecs.a build/cores/VideoPlayer/codecs/dvdcodecs.a build/cores/VideoPlayer/Buffers/videoplayer-buffers.a build/cores/VideoPlayer/VideoPlayer.a build/video/windows/video_windows.a build/video/tags/video_tags.a build/video/jobs/video_jobs.a build/video/guilib/video_guilib.a build/video/dialogs/video_dialogs.a build/video/video.a build/windows/windows.a build/windowing/windowing.a build/weather/weather.a build/view/view.a build/utils_guilib/utils_guilib.a build/utils/utils.a build/threads/threads.a build/storage/storage.a build/rendering/rendering.a build/programs/programs.a build/powermanagement/powermanagement.a build/playlists/playlists.a build/platform/platform_common.a build/pictures_metadata/pictures_metadata.a build/pictures/pictures.a build/messagingHelpers/messagingHelpers.a build/messaging/messaging.a build/imagefiles/imagefiles.a build/guilib_listproviders/guilib_listproviders.a build/guilib_announcement_handlers/guilib_announcement_handlers.a build/guilib_guiinfo/guilib_guiinfo.a build/guilib/guilib.a build/favourites/favourites.a build/dialogs/dialogs.a build/dbwrappers/dbwrappers.a build/commons/commons.a build/application/application.a build/addons_settings/addons_settings.a build/addons_interfaces_gui_dialogs/addons_interfaces_gui_dialogs.a build/addons_interfaces_gui_controls/addons_interfaces_gui_controls.a build/addons_interfaces_gui/addons_interfaces_gui.a build/addons_interfaces/addons_interfaces.a build/addons_gui_skin/addons_gui_skin.a build/addons_gui/addons_gui.a build/addons_binary-addons/addons_binary-addons.a build/addons_addoninfo/addons_addoninfo.a build/addons/addons.a build/xbmc/xbmc.a build/settings/windows/settings_windows.a build/settings/lib/settings_lib.a build/settings/dialogs/settings_dialogs.a build/settings/settings.a build/cores/RetroPlayer/streams/memory/retroplayer_memory.a build/cores/RetroPlayer/streams/retroplayer_streams.a build/cores/RetroPlayer/savestates/retroplayer_savestates.a build/cores/RetroPlayer/rendering/VideoRenderers/rp-videorenderers.a build/cores/RetroPlayer/rendering/rp-rendering.a build/cores/RetroPlayer/process/rp-process.a build/cores/RetroPlayer/playback/retroplayer_playback.a build/cores/RetroPlayer/guiwindows/retroplayer_guiwindows.a build/cores/RetroPlayer/guiplayback/retroplayer_guiplayback.a build/cores/RetroPlayer/guicontrols/retroplayer_guicontrols.a build/cores/RetroPlayer/guibridge/retroplayer_guibridge.a build/cores/RetroPlayer/cheevos/retroplayer_cheevos.a build/cores/RetroPlayer/buffers/video/rp-buffers-video.a build/cores/RetroPlayer/buffers/rp-buffers.a build/cores/RetroPlayer/audio/rp_audio.a build/cores/RetroPlayer/retroplayer.a build/pvr/windows/pvr_windows.a build/pvr/utils/pvr_utils.a build/pvr/timers/pvr_timers.a build/pvr/settings/pvr_settings.a build/pvr/recordings/pvr_recordings.a build/pvr/providers/pvr_providers.a build/pvr/guilib/guiinfo/pvr_guilib_guiinfo.a build/pvr/guilib/pvr_guilib.a build/pvr/filesystem/pvr_filesystem.a build/pvr/epg/pvr_epg.a build/pvr/dialogs/pvr_dialogs.a build/pvr/channels/pvr_channels.a build/pvr/addons/pvr_addons.a build/pvr/pvr.a build/profiles/windows/profiles_windows.a build/profiles/dialogs/profiles_dialogs.a build/profiles/profiles.a build/peripherals/events/peripherals_events.a build/peripherals/dialogs/peripherals_dialogs.a build/peripherals/devices/peripherals_devices.a build/peripherals/bus/virtual/peripheral_bus_virtual.a build/peripherals/bus/peripherals_bus.a build/peripherals/addons/peripherals_addons.a build/peripherals/peripherals.a build/network/websocket/network_websockets.a build/network/network.a build/music/windows/music_windows.a build/music/tags/music_tags.a build/music/jobs/music_jobs.a build/music/infoscanner/music_infoscanner.a build/music/dialogs/music_dialogs.a build/music/music.a build/drm/drm.a build/media/media.a build/interfaces/json-rpc/jsonrpc_interface.a build/interfaces/info/info_interface.a build/interfaces/generic/generic_interface.a build/interfaces/builtins/interfaces_builtins.a build/interfaces/interfaces.a build/input/mouse/generic/input_mouse_generic.a build/input/mouse/input_mouse.a build/input/keymaps/touch/input_keymaps_touch.a build/input/keymaps/remote/input_keymaps_remote.a build/input/keymaps/keyboard/input_keymaps_keyboard.a build/input/keymaps/joysticks/input_keymaps_joysticks.a build/input/keymaps/generic/input_keymaps_generic.a build/input/keymaps/input_keymaps.a build/input/keyboard/generic/input_keyboard_generic.a build/input/keyboard/input_keyboard.a build/input/joysticks/mapping/input_joystick_mapping.a build/input/joysticks/generic/input_joystick_generic.a build/input/joysticks/dialogs/input_joystick_dialogs.a build/input/joysticks/input_joystick.a build/input/actions/input_actions.a build/input/input.a build/games/windows/gameswindows.a build/games/tags/gametags.a build/games/ports/windows/games_ports_windows.a build/games/ports/types/games_ports_types.a build/games/ports/input/games_ports_input.a build/games/ports/guicontrols/games_ports_guicontrols.a build/games/dialogs/osd/gameosddialogs.a build/games/dialogs/gamedialogs.a build/games/controllers/windows/games_controller_windows.a build/games/controllers/types/games_controller_types.a build/games/controllers/listproviders/games_controller_listproviders.a build/games/controllers/input/games_controller_input.a build/games/controllers/guicontrols/games_controller_guicontrols.a build/games/controllers/dialogs/games_controller_dialogs.a build/games/controllers/games_controller.a build/games/agents/windows/games_agents_windows.a build/games/agents/input/games_agents_input.a build/games/addons/streams/game_addon_streams.a build/games/addons/input/gameinput.a build/games/addons/cheevos/gamecheevos.a build/games/addons/gameaddons.a build/games/games.a build/filesystem/videodatabase/videodatabasedirectory.a build/filesystem/musicdatabase/musicdatabasedirectory.a build/filesystem/filesystem.a build/events/windows/events_windows.a build/events/events.a build/cores/playercorefactory/playercorefactory.a build/cores/paplayer/paplayer.a build/cores/externalplayer/externalplayer.a build/cores/dll-loader/exports/util/exports_utils.a build/cores/dll-loader/exports/dllexports.a build/cores/dll-loader/dllloader.a build/cores/audioengine/audioengine.a build/cores/cores.a build/windowing/wayland/windowing_WAYLAND.a build/cores/VideoPlayer/Process/wayland/processWayland.a build/cores/RetroPlayer/process/wayland/rp-process-wayland.a build/upnp/libupnp.a build/swig/libpython_binding.a /usr/lib/x86_64-linux-gnu/libass.so /usr/lib/x86_64-linux-gnu/libcdio.so build/lib/libcrossguid.a /usr/lib/x86_64-linux-gnu/libuuid.so /usr/lib/x86_64-linux-gnu/libcurl.so /usr/lib/x86_64-linux-gnu/libexiv2.so.0.28.5 /usr/lib/x86_64-linux-gnu/libavcodec.so /usr/lib/x86_64-linux-gnu/libavfilter.so /usr/lib/x86_64-linux-gnu/libavformat.so /usr/lib/x86_64-linux-gnu/libavutil.so /usr/lib/x86_64-linux-gnu/libswscale.so /usr/lib/x86_64-linux-gnu/libswresample.so /usr/lib/x86_64-linux-gnu/libpostproc.so -L/usr/lib/x86_64-linux-gnu /usr/lib/x86_64-linux-gnu/libfreetype.so /usr/lib/x86_64-linux-gnu/libfribidi.so /usr/lib/libfstrcmp.so /usr/lib/x86_64-linux-gnu/libharfbuzz.so /usr/lib/x86_64-linux-gnu/liblzo2.so /usr/lib/x86_64-linux-gnu/libssl.so /usr/lib/x86_64-linux-gnu/libcrypto.so /usr/lib/x86_64-linux-gnu/libpcre2-8.so /usr/lib/x86_64-linux-gnu/libspdlog.so.1.15.1 /usr/lib/x86_64-linux-gnu/libfmt.so.10.1.0 /usr/lib/x86_64-linux-gnu/libsqlite3.so /usr/lib/x86_64-linux-gnu/libtag.so.2.0.2 /usr/lib/x86_64-linux-gnu/libtinyxml.so /usr/lib/x86_64-linux-gnu/libtinyxml2.so.10.0.0 /usr/lib/x86_64-linux-gnu/libwayland-client++.so /usr/lib/x86_64-linux-gnu/libwayland-cursor++.so /usr/lib/x86_64-linux-gnu/libwayland-egl++.so /usr/lib/x86_64-linux-gnu/libdrm.so /usr/lib/x86_64-linux-gnu/libxkbcommon.so /usr/lib/x86_64-linux-gnu/libGL.so -lEGL /usr/lib/x86_64-linux-gnu/libshairplay.so /usr/lib/x86_64-linux-gnu/libasound.so /usr/lib/x86_64-linux-gnu/libavahi-client.so /usr/lib/x86_64-linux-gnu/libavahi-common.so /usr/lib/x86_64-linux-gnu/libbluetooth.so /usr/lib/x86_64-linux-gnu/libbluray.so /usr/lib/x86_64-linux-gnu/libcap.so /usr/lib/x86_64-linux-gnu/libcec.so /usr/lib/x86_64-linux-gnu/libp8-platform.so /usr/lib/x86_64-linux-gnu/libdbus-1.so /usr/lib/x86_64-linux-gnu/liblcms2.so /usr/lib/x86_64-linux-gnu/liblirc_client.so /usr/lib/x86_64-linux-gnu/libmicrohttpd.so /usr/lib/x86_64-linux-gnu/libnfs.so /usr/lib/x86_64-linux-gnu/libplist-2.0.so /usr/lib/x86_64-linux-gnu/libpulse.so /usr/lib/x86_64-linux-gnu/libpulse-mainloop-glib.so /usr/lib/x86_64-linux-gnu/libpulse-simple.so /usr/lib/x86_64-linux-gnu/libpython3.13.so /usr/lib/x86_64-linux-gnu/libsmbclient.so /usr/lib/x86_64-linux-gnu/libsndio.so /usr/lib/x86_64-linux-gnu/libudev.so /usr/lib/x86_64-linux-gnu/libudfread.so /usr/lib/x86_64-linux-gnu/libxslt.so /usr/lib/x86_64-linux-gnu/libxml2.so /usr/lib/x86_64-linux-gnu/libva.so /usr/lib/x86_64-linux-gnu/libva-drm.so /usr/lib/x86_64-linux-gnu/libva-wayland.so /usr/lib/x86_64-linux-gnu/libmysqlclient.so -ldl
build/filesystem/filesystem.a(ZipFile.cpp.o):ZipFile.cpp:function XFILE::CZipFile::Close():(.text+0x2ea): error: undefined reference to 'inflateEnd'
build/filesystem/filesystem.a(ZipFile.cpp.o):ZipFile.cpp:function XFILE::CZipFile::Seek(long, int):(.text+0x5c2): error: undefined reference to 'inflateEnd'
build/filesystem/filesystem.a(ZipFile.cpp.o):ZipFile.cpp:function XFILE::CZipFile::Seek(long, int):(.text+0x5db): error: undefined reference to 'inflateInit2_'
build/filesystem/filesystem.a(ZipFile.cpp.o):ZipFile.cpp:function XFILE::CZipFile::~CZipFile():(.text+0x6e2): error: undefined reference to 'inflateEnd'
build/filesystem/filesystem.a(ZipFile.cpp.o):ZipFile.cpp:function XFILE::CZipFile::~CZipFile():(.text+0x772): error: undefined reference to 'inflateEnd'
build/filesystem/filesystem.a(ZipFile.cpp.o):ZipFile.cpp:function XFILE::CZipFile::Read(void*, unsigned long):(.text+0x8d5): error: undefined reference to 'inflate'
build/filesystem/filesystem.a(ZipFile.cpp.o):ZipFile.cpp:function XFILE::CZipFile::Read(void*, unsigned long):(.text+0x96b): error: undefined reference to 'inflate'
build/filesystem/filesystem.a(ZipFile.cpp.o):ZipFile.cpp:function XFILE::CZipFile::InitDecompress():(.text+0xfd9): error: undefined reference to 'inflateInit2_'
build/filesystem/filesystem.a(ZipFile.cpp.o):ZipFile.cpp:function XFILE::CZipFile::DestroyBuffer(void*, int):(.text+0x10f0): error: undefined reference to 'inflate'
build/filesystem/filesystem.a(ZipFile.cpp.o):ZipFile.cpp:function XFILE::CZipFile::UnpackFromMemory(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, bool):(.text+0x1749): error: undefined reference to 'inflateInit2_'
build/filesystem/filesystem.a(ZipFile.cpp.o):ZipFile.cpp:function XFILE::CZipFile::DecompressGzip(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&):(.text+0x1951): error: undefined reference to 'inflateInit2_'
build/filesystem/filesystem.a(ZipFile.cpp.o):ZipFile.cpp:function XFILE::CZipFile::DecompressGzip(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&):(.text+0x1988): error: undefined reference to 'inflate'
collect2: error: ld returned 1 exit status
gmake[2]: *** [CMakeFiles/kodi.dir/build.make:563: kodi-wayland] Error 1
gmake[2]: Leaving directory '/home/defiant/Applications/kodi-build'
gmake[1]: *** [CMakeFiles/Makefile2:6932: CMakeFiles/kodi.dir/all] Error 2
gmake[1]: Leaving directory '/home/defiant/Applications/kodi-build'
gmake: *** [Makefile:146: all] Error 2
```


### Screenshots 
Here are some links or screenshots to help explain the problem:
N/A



## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:
<!--- How has this bug affected you? What were you trying to accomplish? -->
<!--- Put your text below this line -->
apt policy zlib*
```
zlib-bin:
  Installed: (none)
  Candidate: (none)
  Version table:
zlib1g-dev:
  Installed: 1:1.3.dfsg+really1.3.1-1ubuntu1
  Candidate: 1:1.3.dfsg+really1.3.1-1ubuntu1
  Version table:
 *** 1:1.3.dfsg+really1.3.1-1ubuntu1 500
        500 http://gb.archive.ubuntu.com/ubuntu plucky/main amd64 Packages
        100 /var/lib/dpkg/status
zlib1:
  Installed: (none)
  Candidate: (none)
  Version table:
zlib1g:
  Installed: 1:1.3.dfsg+really1.3.1-1ubuntu1
  Candidate: 1:1.3.dfsg+really1.3.1-1ubuntu1
  Version table:
 *** 1:1.3.dfsg+really1.3.1-1ubuntu1 500
        500 http://gb.archive.ubuntu.com/ubuntu plucky/main amd64 Packages
        100 /var/lib/dpkg/status
zlib1-dev:
  Installed: (none)
  Candidate: (none)
  Version table:
```

### Your Environment
Used Operating system:
<!--- Include as many relevant details about the environment you experienced the bug in -->
<!--- Put your text below this line. Checkboxes can easily be ticked once issue is created -->
 - [ ] Android
 - [ ] iOS
 - [ ] tvOS
 - [ x] Linux
 - [ ] macOS
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name: Ubuntu 25.04 amd64
 - Kodi version: git master current



<!--- End of this issue -->
*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

having same issue with x11 , I should have stayed on 24.10 > 25.04 killed everything ... 


``
cmake ../ -DCMAKE_INSTALL_PREFIX=/usr/local -DCORE_PLATFORM_NAME=x11 -DAPP_RENDER_SYSTEM=gl
cmake --build . -- VERBOSE=1 -j$(getconf _NPROCESSORS_ONLN)

L.so /usr/lib/x86_64-linux-gnu/libvdpau.so /usr/lib/x86_64-linux-gnu/libmysqlclient.so -ldl
/usr/bin/ld: build/filesystem/filesystem.a(ZipFile.cpp.o): undefined reference to symbol 'inflateEnd'
/usr/bin/ld: /lib/x86_64-linux-gnu/libz.so.1: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
``

--- a/cmake/modules/FindCurl.cmake
+++ b/cmake/modules/FindCurl.cmake
@@ -22,7 +22,7 @@ if(NOT TARGET ${APP_NAME_LC}::${CMAKE_FIND_PACKAGE_NAME})
     if(NOT CMAKE_SYSTEM_NAME MATCHES "Darwin" AND NOT (WIN32 OR WINDOWS_STORE))
       set(ZLIB_USE_STATIC_LIBS ON)
     endif()
-    find_package(Zlib REQUIRED ${SEARCH_QUIET})
+    find_package(ZLIB REQUIRED ${SEARCH_QUIET})
     unset(ZLIB_USE_STATIC_LIBS)
 
     set(CURL_VERSION ${${${CMAKE_FIND_PACKAGE_NAME}_MODULE}_VER})
@@ -60,14 +60,14 @@ if(NOT TARGET ${APP_NAME_LC}::${CMAKE_FIND_PACKAGE_NAME})
                                                                     LIBRARY::NGHttp2
                                                                     OpenSSL::Crypto
                                                                     OpenSSL::SSL
-                                                                    LIBRARY::Zlib)
+                                                                    LIBRARY::ZLIB)
 
     # Add dependencies to build target
     add_dependencies(${${${CMAKE_FIND_PACKAGE_NAME}_MODULE}_BUILD_NAME} LIBRARY::Brotli
                                                                         LIBRARY::NGHttp2
                                                                         OpenSSL::SSL
                                                                         OpenSSL::Crypto
-                                                                        LIBRARY::Zlib)
+                                                                        LIBRARY::ZLIB)
   endmacro()
 
   # If there is a potential this library can be built internally

--- a/cmake/modules/FindExiv2.cmake
+++ b/cmake/modules/FindExiv2.cmake
@@ -12,7 +12,7 @@ if(NOT TARGET ${APP_NAME_LC}::${CMAKE_FIND_PACKAGE_NAME})
   macro(buildmacroExiv2)
     find_package(Brotli REQUIRED ${SEARCH_QUIET})
     find_package(Iconv REQUIRED ${SEARCH_QUIET})
-    find_package(Zlib REQUIRED ${SEARCH_QUIET})
+    find_package(ZLIB REQUIRED ${SEARCH_QUIET})
 
     # Patch pending review upstream (https://github.com/Exiv2/exiv2/pull/3004)
     set(patches "${CMAKE_SOURCE_DIR}/tools/depends/target/${${CMAKE_FIND_PACKAGE_NAME}_MODULE_LC}/0001-WIN-lib-postfix.patch")
@@ -56,7 +56,7 @@ if(NOT TARGET ${APP_NAME_LC}::${CMAKE_FIND_PACKAGE_NAME})
     # Link libraries for target interface
     set(${${CMAKE_FIND_PACKAGE_NAME}_MODULE}_LINK_LIBRARIES LIBRARY::Brotli
                                                             LIBRARY::Iconv
-                                                            LIBRARY::Zlib)
+                                                            LIBRARY::ZLIB)
 
     if(CORE_SYSTEM_NAME STREQUAL "freebsd")
       list(APPEND ${${CMAKE_FIND_PACKAGE_NAME}_MODULE}_LINK_LIBRARIES procstat)
@@ -67,7 +67,7 @@ if(NOT TARGET ${APP_NAME_LC}::${CMAKE_FIND_PACKAGE_NAME})
     # Add dependencies to build target
     add_dependencies(${${${CMAKE_FIND_PACKAGE_NAME}_MODULE}_BUILD_NAME} LIBRARY::Brotli
                                                                         LIBRARY::Iconv
-                                                                        LIBRARY::Zlib)
+                                                                        LIBRARY::ZLIB)
   endmacro()
 
   include(cmake/scripts/common/ModuleHelpers.cmake)
@@ -79,7 +79,7 @@ if(NOT TARGET ${APP_NAME_LC}::${CMAKE_FIND_PACKAGE_NAME})
     # Dependency list of this find module for an INTERNAL build
     set(${CMAKE_FIND_PACKAGE_NAME}_DEPLIST Brotli
                                            Iconv
-                                           Zlib)
+                                           ZLIB)
 
     check_dependency_build(${CMAKE_FIND_PACKAGE_NAME} "${${CMAKE_FIND_PACKAGE_NAME}_DEPLIST}")
   endif()

--- a/cmake/modules/FindLibZip.cmake
+++ b/cmake/modules/FindLibZip.cmake
@@ -22,7 +22,7 @@ find_package(libzip ${CONFIG_${CMAKE_FIND_PACKAGE_NAME}_FIND_SPEC} CONFIG ${SEAR
 if(libzip_VERSION VERSION_LESS ${${${CMAKE_FIND_PACKAGE_NAME}_MODULE}_VER})
   # Check for dependencies
   find_package(GnuTLS REQUIRED ${SEARCH_QUIET})
-  find_package(Zlib REQUIRED ${SEARCH_QUIET})
+  find_package(ZLIB REQUIRED ${SEARCH_QUIET})
 
   # Eventually we will want Find modules for the following deps
   # bzip2 
@@ -38,9 +38,9 @@ if(libzip_VERSION VERSION_LESS ${${${CMAKE_FIND_PACKAGE_NAME}_MODULE}_VER})
   BUILD_DEP_TARGET()
 
   # Todo: Gnutls dependency
-  add_dependencies(${${${CMAKE_FIND_PACKAGE_NAME}_MODULE}_BUILD_NAME} LIBRARY::Zlib)
+  add_dependencies(${${${CMAKE_FIND_PACKAGE_NAME}_MODULE}_BUILD_NAME} LIBRARY::ZLIB)
 
-  set(${${CMAKE_FIND_PACKAGE_NAME}_MODULE}_LINK_LIBRARIES LIBRARY::Zlib)
+  set(${${CMAKE_FIND_PACKAGE_NAME}_MODULE}_LINK_LIBRARIES LIBRARY::ZLIB)
 else()
   # we only do this because we use find_package_handle_standard_args for config time output
   # and it isnt capable of handling TARGETS, so we have to extract the info
@@ -83,7 +83,7 @@ if(LibZip_FOUND)
     # BZip2::BZip2, LibLZMA::LibLZMA, GnuTLS::GnuTLS, Nettle::Nettle
     # For now, we just override 
     set_target_properties(libzip::zip PROPERTIES
-                                      INTERFACE_LINK_LIBRARIES "LIBRARY::Zlib")
+                                      INTERFACE_LINK_LIBRARIES "LIBRARY::ZLIB")
   else()
     SETUP_BUILD_TARGET()
 

--- a/cmake/modules/FindTagLib.cmake
+++ b/cmake/modules/FindTagLib.cmake
@@ -46,9 +46,9 @@ if(NOT TARGET ${APP_NAME_LC}::${CMAKE_FIND_PACKAGE_NAME})
   
     BUILD_DEP_TARGET()
   
-    add_dependencies(${${${CMAKE_FIND_PACKAGE_NAME}_MODULE}_BUILD_NAME} LIBRARY::Zlib
+    add_dependencies(${${${CMAKE_FIND_PACKAGE_NAME}_MODULE}_BUILD_NAME} LIBRARY::ZLIB
                                                                         LIBRARY::Utfcpp)
-    set(${${CMAKE_FIND_PACKAGE_NAME}_MODULE}_LINK_LIBRARIES "LIBRARY::Zlib")
+    set(${${CMAKE_FIND_PACKAGE_NAME}_MODULE}_LINK_LIBRARIES "LIBRARY::ZLIB")
   endmacro()
 
   # If there is a potential this library can be built internally

--- a/cmake/modules/FindZLIB.cmake
+++ b/cmake/modules/FindZLIB.cmake
@@ -6,7 +6,7 @@
 #
 #   ${APP_NAME_LC}::ZLIB - Alias to ZLIB::ZLIB target
 #   LIBRARY::ZLIB - Alias to ZLIB::ZLIB target
-#   ZLIB::ZLIB - standard Zlib target from system find package
+#   ZLIB::ZLIB - standard ZLIB target from system find package
 #
 
 if(NOT TARGET ${APP_NAME_LC}::${CMAKE_FIND_PACKAGE_NAME})
@@ -46,7 +46,7 @@ if(NOT TARGET ${APP_NAME_LC}::${CMAKE_FIND_PACKAGE_NAME})
     mark_as_advanced(ZLIB_FOUND)
   else()
     if(ZLIB_FIND_REQUIRED)
-      message(FATAL_ERROR "Zlib libraries were not found.")
+      message(FATAL_ERROR "ZLIB libraries were not found.")
     endif()
   endif()
 endif()

